src/app/redux/reducers/tableView.js

```diff
@@ -309,10 +305,7 @@ export default (state = initialState, action, completeState) => {
     case SET_CURRENT_TABLE:
       return { ...state, currentTable: action.tableId };
     case COLUMNS_DATA_LOADED:
-      return f.compose(
-        setInitialVisibleColumns(action),
-        setInitialColumnOrdering(action)
-      )(state);
+      return setInitialColumnState(state, action, completeState);
```

Comment:
Gibt es einen Grund dafür, dass beide Dinge jetzt in eine Funktion gemergt wurden? Das zwingt uns ja auch quasi dazu, in Zeilen 173,174 diesen `&&`-Operator mit seinen verwirrenden Effekten zu verwenden.